docket/plugins/config_scripts.py

```diff
@@ -16,13 +15,27 @@
 from docket.plugins._checks import has_guild_perms
 from docket.plugins._group import docket_group
 
+if TYPE_CHECKING:
+    from docket.bot import Docket
+
 plugin = crescent.Plugin(
     "config-scripts", command_hooks=[has_guild_perms(hikari.Permissions.MANAGE_GUILD)]
 )
 scripts = docket_group.sub_group("scripts", "Manage scripts")
 
 
-CODE_BLOCK = re.compile(r"^`*\w*?\s*(?P<code>.+)\s*`*$")
+async def _get_lua_from_message(msg: hikari.Message) -> str:
+    if not msg.content:
+        attachment = msg.attachments[0]
+        return str((await attachment.read()).decode("utf-8"))
+    elif not msg.attachments:
+        lines = msg.content.split("\n")
+        if lines[0].strip() in {"```", "```lua"}:
```

Comment:
Maybe it should support all languages here?

Reply:
I don't think so. Discord considers this valid:
```
```print("hi")```
```

But if we tried to support all languages, the actual code would disappear.

Reply:
You could use regex to replace the pattern `^\`\`\`.+$`. Thats kinda overkill and everything would be messy imo so I think it should only match lua.

Reply:
I don't see any reason for someone to post it with a different language anyways.